gen3/bin/workon.sh

```diff
@@ -113,7 +113,7 @@ if [[ ! -f "$bucketCheckFlag" && "$GEN3_FLAVOR" == "AWS" ]]; then
   }
 EOM
 )
-    gen3_aws_run aws s3api create-bucket --acl private --bucket "$GEN3_S3_BUCKET"
+    gen3_aws_run aws s3api create-bucket --acl private --bucket "$GEN3_S3_BUCKET" --create-bucket-configuration LocationConstraint="$(aws configure get $GEN3_PROFILE.region)"
```

Comment:
It would be cleaner to set this here: https://github.com/uc-cdis/cloud-automation/blob/master/gen3/lib/aws.sh#L15 

```
export AWS_DEFAULT_REGION=$(aws configure get "${AWS_PROFILE}.region" || echo us-east-1)
```

But this would require more granular testing to see if it doesn't break existing setups.